scheduler/test/e2e/scheduler_test.go

```diff
@@ -109,8 +108,7 @@ var _ = Describe("[E2E] KeptnScheduler", Ordered, func() {
 		)
 		BeforeEach(func() {
 			DeferCleanup(func() {
-				err := k8sClient.Delete(ctx, pod)
-				Expect(err).NotTo(HaveOccurred(), "could not remove pod")
+				k8sClient.Delete(ctx, pod)
```

Comment:
Why did you remove the assertions here?

Reply:
We discussed that no Delete failure should be checked since this would make the whole suit fail, aka not cleaning up the rest. Also we ignore already existing errors at creation for the same reason.